package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "videojs-contrib-media-sources",
-  "version": "4.6.2",
+  "version": "4.6.3",
```

Comment:
Usually we change the version as part of a versioning process (outside of the PR)

Reply:
I see. Let me fix this and create another PR.